### PR TITLE
Fixed nanomq rpm package filesystem conflict.

### PR DESCRIPTION
**Pull request by lee-emqx**

### CODE REVIEW

File: .github/workflows/build_packages.yaml [hunk 1]
```diff
@@ -55,26 +57,30 @@ jobs:
         make
     - name: build deb
```
Comment (Rory-Z) [inline .github/workflows/build_packages.yaml:58]:
```suggestion
    - name: build deb
      if: matrix.arch[0] == 'amd64'
```

**Review by Rory-Z — COMMENTED** [review]
Shall we add arm64 and aarch64?

File: .github/workflows/build_packages.yaml (hunk 1 above)
Comment (Rory-Z) [reply .github/workflows/build_packages.yaml:58]: And delete "if" in "run"

Comment (lee-emqx) [reply .github/workflows/build_packages.yaml:58]: Are arm64 and aarch64 the same thing?

Comment (JaylinYu) [reply .github/workflows/build_packages.yaml:58]: correct! and there is a grammar error

**Review by JaylinYu — APPROVED** [review]

**Pull request merged**